src/ppc64.h

```diff
@@ -45,18 +45,40 @@
 #define DES_COPY			0
 #define DES_BS_ASM			0
 #define DES_BS				1
-#define DES_BS_VECTOR			0
+#if defined(JOHN_ALTIVEC)
+#define DES_BS_EXPAND			3
```

Comment:
`DES_BS_EXPAND` must be 0 or 1, never 3. We use it as a Boolean.

Reply:
That was my mistake, taking it from ppc64alt.h